ClothesApp/Infrastructure/Repositories/BrandsRepository.cs

```diff
@@ -0,0 +1,64 @@
+using Application.Repositories;
+using Domain.Entities;
+using Microsoft.EntityFrameworkCore;
+
+namespace Infrastructure.Repositories;
+
+public class BrandsRepository : IBrandsRepository
+{
+    private readonly ClothesAppContext _dbContext;
+
+    public BrandsRepository(ClothesAppContext context)
+    {
+        _dbContext = context;
+    }
+
+    public async Task<Brand> Insert(Brand brand)
+    {
+        await _dbContext.Brands.AddAsync(brand);
```

Comment:
Add should be used instead: https://learn.microsoft.com/en-us/dotnet/api/microsoft.entityframeworkcore.dbcontext.addasync?view=efcore-7.0#microsoft-entityframeworkcore-dbcontext-addasync-1(-0-system-threading-cancellationtoken)
See Remarks